website/docs/r/pod.html.markdown

```diff
@@ -91,7 +96,7 @@ The following arguments are supported:
 
 * `args` - (Optional) Arguments to the entrypoint. The docker image's CMD is used if this is not provided. Variable references $(VAR_NAME) are expanded using the container's environment. If a variable cannot be resolved, the reference in the input string will be unchanged. The $(VAR_NAME) syntax can be escaped with a double $$, ie: $$(VAR_NAME). Escaped references will never be expanded, regardless of whether the variable exists or not. Cannot be updated. For more info see [Kubernetes reference](http://kubernetes.io/docs/user-guide/containers#containers-and-commands)
 * `command` - (Optional) Entrypoint array. Not executed within a shell. The docker image's ENTRYPOINT is used if this is not provided. Variable references $(VAR_NAME) are expanded using the container's environment. If a variable cannot be resolved, the reference in the input string will be unchanged. The $(VAR_NAME) syntax can be escaped with a double $$, ie: $$(VAR_NAME). Escaped references will never be expanded, regardless of whether the variable exists or not. Cannot be updated. For more info see [Kubernetes reference](http://kubernetes.io/docs/user-guide/containers#containers-and-commands)
-* `env` - (Optional) List of environment variables to set in the container. Cannot be updated.
+* `env` - (Optional) Struct of string name and value pairs to set in the container's environment. May be declared multiple times. Cannot be updated.
```

Comment:
I believe HCL calls these constructs "blocks" so I'd be more comfortable if we followed that convention. Can we swap struct for block?
```suggestion
* `env` - (Optional) Block of string name and value pairs to set in the container's environment. May be declared multiple times. Cannot be updated.
```

Reply:
Block sounds good, was looking for examples of similar language in this provider and couldn't find any to reference :)

Re: nested structures: length of documentation aside, I was more thinking that many providers treat 'volume attachment' as a separate resource from the volume declaration. It just so happens that this would flatten the configuration (and documentation) a little bit, I'm happy to do that refactor, but totally agree on avoiding it until after 0.12 is finalized.